bears/js/JSStandardBear.py

```diff
@@ -16,13 +18,53 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1'),
+                    NpmRequirement('eslint-plugin-markdown', '1.0.0-beta.6'),
+                    NpmRequirement('eslint-plugin-html', '3.2.2')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    def create_arguments(self, filename, file, config_file,
+                         language: language = language('JavaScript'),
+                         global_vars: typed_list(str) = (),
+                         eslint_env: typed_list(str) = (),
+                         ):
+        """
+        :param language:
+            Set to ``Javascript`` by default. To check javascript in either
+            markdown or html specify ``Markdown`` or ``html``. Use
+            ``Typescript`` to lint typescript.
+        :param global_vars:
+            The list of global variables in the code
+        :param eslint_env:
+            To support global variables of certain environments. The complete
+            list of supported environments can be found here
+            https://eslint.org/docs/user-guide/configuring#specifying-environments
+        """
+        args = ('--verbose', filename,)
+
+        if 'Markdown' in language:
+            args += ('--plugin', 'markdown',)
+        elif 'Typescript' in language:
+            args += ('--parser', 'typescript-eslint-parser',
+                     '--plugin', 'typescript')
+        elif 'HTML' in language:
+            args += ('--plugin', 'html')
+        elif 'Javascript' not in language:
+            self.err(
+                'Language needs to be either Markdown, HTML, Typescript '
+                'or Javascript. Assuming Javascript.')
```

Comment:
I want to give a review after your decision toward @jayvdb [review](https://github.com/coala/coala-bears/pull/2230/files#r204232076). However, sorry for bring this up, could you write a proper name? Like, change `Javascript` to `JavaScript` instead. This also applies to TypeScript. All of them please.